python/rikai/types/geometry.py

```diff
@@ -281,6 +281,8 @@ def iou(
             other_arr = np.array([other])
         else:
             other_arr = np.array(other)
+        if other_arr.size == 0:
+            return np.zeros(0)
```

Comment:
what's the difference between np.zeros(0) and np.empty?

Reply:
`np.empty` doesn't initialize the array elements to anything. Since in this case there are no array elements it doesn't matter, but I think `np.empty(0)` is cleaner.

Reply:
:this: